extension.json

```diff
@@ -0,0 +1,50 @@
+{
+	"@Note": "When updating this file please also update SemanticForms.php with the same changes.",
+	"load_composer_autoloader": true,
+	"name": "Semantic Forms Select",
+	"version": "1.3.0",
+	"author": [
+		"Jason Zhang",
+		"Toni Hermoso Pulido",
+		"..."
+	],
+	"url": "https://www.mediawiki.org/wiki/Extension:SemanticFormsSelect",
+	"descriptionmsg": "semanticformsselect-desc",
+	"type": "semantic",
+	"license-name": "GPL-2.0+",
+	"callback": "SemanticFormsSelectHooks::onCallback",
+	"ExtensionFunctions": [
+		"SemanticFormsSelectHooks::onExtensionFunction"
+	],
+	"AutoloadClasses": {
+		"SemanticFormsSelectHooks": "SemanticFormsSelect.hooks.php"
```

Comment:
Autoloading is done by Composer, no way manual class registration starts to creep in.